rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -7,9 +7,12 @@ use self::tables::session_space::{ClusterRef, SessionSpace, SessionSpaceRef, Ses
 use self::tables::session_space_normalizer::SessionSpaceNormalizer;
 use id_types::*;
 
+/// The reserved value for an unknown token index.
+/// Used in interop.
 pub const NIL_TOKEN: i64 = -1;
 
 #[derive(Debug)]
+/// The core allocator.
```

Comment:
I'd say something more like "A distributed ID allocator."

And possibly add some example usages that we can show off.